site/assets/css/header.scss

```diff
@@ -10,6 +10,49 @@
   }
 }
 
+.nav-left {
+  overflow: visible;
+  overflow-x: visible;
+  overflow-y: visible;
```

Comment:
These were needed, otherwise the popup wasn't visible at all under the nav header.